brianlong/www.validators.app
# 184549107 - tweak calculating epoch duration

#### What's this PR do?
Move epoch duration value into db and its related logic into a separated service.

#### How should this be manually tested?
- `rails db:migration`
- run `ClusterStats::UpdateEpochDurationService.new(network).call` for each network you want to test. Check and confirm that epoch_duration in ClusterStat was populated with some value
- run `rails r script/gather_stake_accounts.rb` and check for any errors, check pools APY/ROI
- run gather scripts and confirm that there're no errors in sidekiq, and gross ROI is updated with some value

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/184549107)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
